### PR TITLE
react-native: NetInfo.isConnectionExpensive is a function

### PR DESCRIPTION
**Pull request by Skn0tt**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://facebook.github.io/react-native/docs/netinfo#isconnectionexpensive
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/react-native/index.d.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@
 // Definitions by: Eloy Durán <https://github.com/alloy>
 //                 HuHuanming <https://github.com/huhuanming>
 //                 Kyle Roach <https://github.com/iRoachie>
+//                 Simon Knott <https://github.com/skn0tt>
```
Comment (alloy) [inline types/react-native/index.d.ts]: Are you aware that this means to sign up for future PR reviews?

**Review by alloy — COMMENTED** [review]
The change looks good to me, just waiting to hear about the maintainership.

File: types/react-native/index.d.ts (hunk 1 above)
Comment (Skn0tt) [reply types/react-native/index.d.ts]: Yep. I'd be happy to help the TS community! :)

**Comment by alloy** [comment]
Thanks!

**Pull request merged**